node-webservice/react-client/src/Components/TextInput.jsx

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
 
 const inputStyle = {
-    width: '25vw',
+    width: 'calc((10px + 2vmin)*5)',
```

Comment:
<img width="632" alt="Screen Shot 2020-03-31 at 5 47 57 PM" src="https://user-images.githubusercontent.com/11395364/78088022-c9abbf80-7377-11ea-9ff7-d941aff57233.png">

It looks like this change has made the input field shorter and is cutting off the end of the placeholder text.
Not sure if this is intentional, or what the intended effect of this change is

Reply:
The change was to more closely align to how the font size changes with window size.  Originally width was static.  Obviously I made the new width too small, but it does now hold steady with font size.  I'll change it to be a bit wider.